core/src/engine.cpp

```diff
@@ -4,18 +4,28 @@
 namespace knot {
 
 Engine::Engine() {
+    InitializeYojimbo();
+
     m_framebufferManager = std::make_shared<knot::FramebufferManager>(*this);
     m_windowModule = std::make_shared<knot::Window>(m_windowWidth, m_windowHeight, m_windowTitle, *this);
     m_forwardRenderModule = std::make_shared<knot::ForwardRenderer>(*this);
     m_physicsModule = std::make_shared<knot::Physics>(*this);
     m_assetManager = std::make_shared<knot::AssetManager>();
+    if (!isClient) {
+        m_serverModule = std::make_shared<knot::NetworkedServer>(*this);
+    }
```

Comment:
should consider having a different build for server and using the 
m_engine->add_subsystem(m_serverModule)
an example of it being use is with the widget manager
so that we do not need to have a conditional inside of Engine::Engine()

Reply:
I would argue that having the same build for both client and server is beneficial in that we only have to build, and share 1 build, and anybody can be a server

Reply:
I had it as this yesterday with two branches for server and client builds, it was decided that we would try and unify the branches and be able to set the arguments in main for deciding whether it is server or client